plugin/pom.xml

```diff
@@ -99,6 +99,11 @@
       <version>0.50.40</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.vaadin.external.google</groupId>
+      <artifactId>android-json</artifactId>
```

Comment:
where has this came from?

Reply:
Strange sounds like it imported a different json